.github/workflows/test.yaml

```diff
@@ -38,6 +38,12 @@ jobs:
           extra-specs: |
             python=${{ matrix.python-version }}
 
+      - name: Upstream packages
+        if: ${{ matrix.python-version == '3.11' }}
+        run: |
+          python -m pip install git+https://github.com/dask/distributed
+          python -m pip install git+https://github.com/dask/dask
```

Comment:
#156 was recently merged, and it now installs dask dev from source in `environment.yml`. Should we also add distributed to that file?

Reply:
Yeah, I think so. Let's close this PR in favor of https://github.com/dask-contrib/dask-expr/pull/190